packaging/rpm/microshift.spec

```diff
@@ -464,6 +477,13 @@ mkdir -p -m755 %{buildroot}%{_datadir}/microshift/release
 install -p -m644 assets/optional/flannel/release-flannel-{x86_64,aarch64}.json %{buildroot}%{_datadir}/microshift/release/
 %endif
 
+%if %{with_topolvm}
+# topolvm
+install -d -m755 %{buildroot}/%{_prefix}/lib/microshift/manifests.d/001-microshift-topolvm
+install -p -m644 assets/optional/topolvm/*.yaml %{buildroot}/%{_prefix}/lib/microshift/manifests.d/001-microshift-topolvm
+install -p -m644 packaging/topolvm/00-disableDefaultCSI.yaml %{buildroot}%{_sysconfdir}/microshift/config.d/01-disableDefaultCSI.yaml
```

Comment:
What do you think about using the [packaging/microshift/dropins/](https://github.com/openshift/microshift/tree/main/packaging/microshift/dropins) dir instead of creating `packaging/topolvm/`?

Reply:
sure